src/main/webapp/i18n/en/global.json

```diff
@@ -278,7 +278,8 @@
             "pattern": "This field must fulfill the pattern {{ pattern }}.",
             "number": "This field must be a number.",
             "datetimelocal": "This field must be a date and time.",
-            "patternLogin": "This field may only contain letters, digits and e-mail addresses."
+            "patternLogin": "This field may only contain letters, digits and e-mail addresses.",
+            "validPattern": "This field contains an invalid pattern"
```

Comment:
why is it called validPattern if it is about containing an invalid pattern?
Should it end with "." as the other messages?
Or what I would find better "This field does not match the given pattern." As you either define valid patterns that must be included instead of invalid patterns that are checked against.

Reply:
Thanks @lschlesinger for the suggestion :slightly_smiling_face: 
You'are right, 'validPattern' might be somehow confusing, but would leave the text as is, as there are no explicitly 'given patterns'. Only either valid or invalid (RegEx) patterns IMO...

Do you think that replacing 'pattern' with 'regular expression pattern' would be more understandable?